automation/cli.py

```diff
@@ -57,18 +61,67 @@ def exist_directory(x: str) -> str:
         raise argparse.ArgumentTypeError("%s isn't a directory" % (x,))
     return x
 
+def parse_receiver_job(job) -> Tuple[str, datetime.datetime, datetime.datetime]:
+    '''
+    Parse receive job CRON entry.
```

Comment:
```suggestion
    Parse reception job CRON entry.
```

Reply:
Do you want to replace "receive" with "reception" in all places in project?